12.0/base_requirements.txt

```diff
@@ -28,7 +28,6 @@ pyserial==3.1.1
 python-dateutil==2.5.3
 pytz==2016.7
 pyusb==1.0.0
-PyYAML==3.12 ; python_version < '3.7'
```

Comment:
PyYaml has been dropped by odoo https://github.com/odoo/odoo/commit/a277bfb44a725f201b728dfc79515a01c0a29f6a#diff-b4ef698db8ca845e5845c4618278f29a

AFAIR it can still be used in OCA modules and we use it in our travis. So I just moved it to the Library dependencies